ajax/libs/labelauty/1.1.0/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "homepage": "https://github.com/fntneves/jquery-labelauty",
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.